pybossa/view/admin.py

```diff
@@ -388,8 +386,8 @@ def dashboard():
                                returning_users_week=returning_users_week,
                                update_feed=update_feed,
                                wait=False)
-    except ProgrammingError:
-        db.slave_session.rollback()
+    except Exception as e:
```

Comment:
This is too generic :-) I would recommend to avoid Bare Exceptions and use the ProgrammingError one to be logged out. If something else is happening, then the server should return a 500, and we'll be able to debug it :-)